app/sa/controllers/testOnly/AtsSaFieldListController.scala

```diff
@@ -167,13 +167,37 @@ class AtsSaFieldListController @Inject() (
 
   private val fields2024: Seq[String] = fields2023 :+ "ctnTaxOnTransitionPrft"
 
+  private val fields2025: Seq[String] = fields2024 ++ Seq(
+    "capOffshoreTrustLiability",
+    "incomeTermination",
+    "taxablePaySar",
+    "taxableRedundancySar",
+    "taxOnPaySar",
+    "taxOnRedundancySar",
+    "cgAtHigherRateCi",
```

Comment:
These fields need to be exactly as listed in the matrix spreadsheet. For example, it should be  "cGAtHigherRateCI" not "cgAtHigherRateCi" - the case has to be exactly as in the matrix spreadsheet. You will see errors in the sa uprating tool when you paste in the "Data name test case" columns into the tool: 
"Unrecognised field names: cGAtHigherRateCI, cGAtHigherRateRP, cGAtLowerRateCI, cGAtLowerRateRP, higherRateCgtCI, higherRateCgtRP, incomeTaxStatus, lowerRateCgtCI, lowerRateCgtRP, taxOnTransitionPrft"